aiida/manage/external/rmq.py

```diff
@@ -212,6 +212,20 @@ async def _continue(self, communicator, pid, nowait, tag=None):
             message = 'the class of the process could not be imported.'
             self.handle_continue_exception(node, exception, message)
             raise
+        except DuplicateSubscriberIdentifier:
+            # This will happen when a subscriber with this process identifier is already registered. The reasons could
+            # be the user mistakenly recreating the task, thinking it was lost, or the task got automatically requeued
+            # by RabbitMQ because the daemon worker that was working on it, did not respond to the heartbeat in time,
+            # causing RabbitMQ to assume it had died. In any case, in most cases, the process is actually already being
+            # run by another worker and we should not let it except here. Instead we log the message and hope that the
+            # original subsriber will properly bring the process to a terminal state. In case that the task actually got
+            # lost and the duplicate subscriber simply hadn't been properly cleaned, a user can always recreate the task
+            # in the future once the subscriber has been cleared.
```

Comment:
```suggestion
            # This happens when a subscriber with this process identifier is already registered. Possible reasons include:
            #  1. The user mistakenly recreates the corresponding task, thinking the original task was lost.
            #  2. RabbitMQ automatically requeues the task because the daemon worker assigned to it does not respond to the heartbeat in time, causing RabbitMQ to assume it had died. 
            # Here we assume that the existence of another subscriber indicates that the process is still being
            # run by a worker. We thus ignore the request to have another worker take it on and
            # consume the current task from the queue (`return False`). 
            # If our assumption was wrong and the original task was no longer being worked on, the user can resubmit the task once the list of subscribers of the process has been cleared.
            # Note: In case 2 we are deleting the *original* task, and once the original worker finishes running the process, there won't be a task in RabbitMQ to acknowledge anymore. This, however, is silently ignored.
```